tests/node/rpc/messages/UnsubscribeVehicleDataResponseTests.js

```diff
@@ -33,13 +39,18 @@ describe('UnsubscribeVehicleDataResponseTests', function () {
             .setDataType(VehicleDataType.VEHICLEDATA_WINDOWSTATUS);
         const JSON_WINDOWSTATUS = this.windowStatus.getParameters();
 
+        this.climateData = new VehicleDataResult()
+          .setDataType(VehicleDataType.VEHICLEDATA_CLIMATEDATA);
```

Comment:
The indentation for line 43 is incorrect. It has an indentation of 10 spaces but it should be 12 spaces.